packages/angular-cli/commands/build.ts.orig

```diff
@@ -0,0 +1,91 @@
+const Command = require('../ember-cli/lib/models/command');
+import WebpackBuild from '../tasks/build-webpack';
+import WebpackBuildWatch from '../tasks/build-webpack-watch';
+
+export interface BuildOptions {
+  target?: string;
+  environment?: string;
+  outputPath?: string;
+  watch?: boolean;
+  watcher?: string;
+  supressSizes: boolean;
+  baseHref?: string;
+  aot?: boolean;
+<<<<<<< HEAD
```

Comment:
Woops, I'm sorry these .orig files should never have been committed. 

These are artifacts from merging and shouldn't have been added I'm used to them being ignored. 

Anyway I Removed them and everything should be good.